controllers/openshift_controller_test.go

```diff
@@ -631,6 +631,42 @@ var _ = Describe("OpenShift KataConfig Controller", func() {
 			By("Creating the KataConfig CR successfully")
 			Expect(k8sClient.Create(context.Background(), kataConfig)).Should(Succeed())
 
+			// Delete
+			By("Deleting KataConfig CR successfully")
+			kataConfigKey := types.NamespacedName{Name: kataconfig.Name}
+			Eventually(func() error {
+				k8sClient.Get(context.Background(), kataConfigKey, kataconfig)
+				return k8sClient.Delete(context.Background(), kataconfig)
+			}, timeout, interval).Should(Succeed())
+
+			By("Expecting to delete KataConfig CR successfully")
+			Eventually(func() error {
+				return k8sClient.Get(context.Background(), kataConfigKey, kataconfig)
+			}, timeout, interval).ShouldNot(Succeed())
+
+		})
+	})
+	Context("Custom KataConfig with CheckNodeEligibility and PeerPods enabled", func() {
+		It("Should ignore CheckNodeEligibility when PeerPods is enabled", func() {
+
+			kataConfig := &kataconfigurationv1.KataConfig{
+				TypeMeta: metav1.TypeMeta{
+					APIVersion: "kataconfiguration.openshift.io/v1",
+					Kind:       "KataConfig",
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Name: name + "with-peerpods",
```

Comment:
```suggestion
					Name: name + "-with-peerpods",
```
This will produce a prettier `example-kataconfig-with-peerpods` name :smiley: